project-words.txt

```diff
@@ -330,8 +330,11 @@ showinfo
 Sida
 Simen
 slorber
+Slorber
 sluggifies
 sluggify
+socialnull
+socialnumber
```

Comment:
Try to get rid of these by using camel case `socialNull` for example, both words exist independently